examples/vision/depth_estimation.py

```diff
@@ -0,0 +1,507 @@
+"""
+Title: An approach towards Depth Estimation with CNN
+Author: [Victor Basu](https://www.linkedin.com/in/victor-basu-520958147)
+Date created: 2021/08/09
+Last modified: 2021/08/10
+Description: Implement an depth estimation model with CNN.
```

Comment:
please use `a depth` instead of `an depth`

Reply:
Please remove full stop from end of line

Reply:
updated